src/zocalo/cli/configure_rabbitmq.py

```diff
@@ -359,31 +364,36 @@ def run():
     with open(args.configuration) as in_file:
         yaml_data = yaml.safe_load(in_file)
 
-    if args.user_config:
-        _configure_users(api, args.user_config)
-    _configure_policies(api, yaml_data["policies"])
-
-    queue_specs = []
-    exchange_specs = get_exchange_specs(yaml_data["exchanges"])
-    binding_specs = get_binding_specs(yaml_data.get("bindings", []))
-    for group in yaml_data["groups"]:
-        if group.get("settings", {}).get("broadcast"):
-            exchange_specs.extend(get_exchange_specs_for_group(group))
-        else:
-            queue_specs.extend(get_queue_specs(group))
-            binding_specs.extend(get_binding_specs_for_group(group))
-
-    _configure_queues(api, queue_specs)
-    update_config(api, exchange_specs, api.exchanges())
-    permanent_bindings = []
-    # don't remove bindings to temporary queues
-    queues = api.queues()
-    for b in api.bindings():
-        q = [qu for qu in queues if qu.vhost == b.vhost and qu.name == b.destination][0]
-        if q.auto_delete or q.exclusive:
-            continue
-        permanent_bindings.append(b)
-    update_config(api, binding_specs, permanent_bindings)
+    try:
+        if args.user_config:
+            _configure_users(api, args.user_config)
+        _configure_policies(api, yaml_data["policies"])
+
+        queue_specs = []
+        exchange_specs = get_exchange_specs(yaml_data["exchanges"])
+        binding_specs = get_binding_specs(yaml_data.get("bindings", []))
+        for group in yaml_data["groups"]:
+            if group.get("settings", {}).get("broadcast"):
+                exchange_specs.extend(get_exchange_specs_for_group(group))
+            else:
+                queue_specs.extend(get_queue_specs(group))
+                binding_specs.extend(get_binding_specs_for_group(group))
+
+        _configure_queues(api, queue_specs)
+        update_config(api, exchange_specs, api.exchanges())
+        permanent_bindings = []
+        # don't remove bindings to temporary queues
+        queues = api.queues()
+        for b in api.bindings():
+            q = [
+                qu for qu in queues if qu.vhost == b.vhost and qu.name == b.destination
+            ][0]
+            if q.auto_delete or q.exclusive:
+                continue
+            permanent_bindings.append(b)
+        update_config(api, binding_specs, permanent_bindings)
+    except requests.exceptions.HTTPError as e:
+        logger.error(e)
```

Comment:
Do we want `exc_info=True`?

Reply:
I thought that in the context of the configuration script that the traceback didn't really add any additional information - the important details come in the error message returned by the API. E.g. attempting to create a binding between a queue `bar` and non-existent exchange `foo` gives:
```
$ zocalo.configure_rabbitmq /path/to/rabbitmq-configuration.yaml 
Creating queue: name='bar' durable=True auto_delete=False arguments={'x-queue-type': 'quorum', 'x-single-active-consumer': False} vhost='zocalo'
creating BindingSpec source='foo' destination='bar' destination_type=<DestinationType.QUEUE: 'q'> vhost='zocalo' routing_key='bar' arguments={}
404 Client Error: Not Found for url: http://example.com:15672/api/bindings/zocalo/e/foo/q/bar
```
I'm not convinced that adding the relevant traceback adds any useful information here...

Reply:
Yeah, ok. If you think it won't add anything that's fine